dcpquery/_config.py

```diff
@@ -72,7 +72,7 @@ def db(self):
                 db_host = AwsSecret(db_host_secret_name).value.strip()
             db_name = self.app_name
             self._db = sqlalchemy.create_engine(f"postgresql+psycopg2://{db_user}:{db_password}@{db_host}/{db_name}",
-                                                **self._db_engine_params)
+                                                implicit_returning=False, **self._db_engine_params)
```

Comment:
This should be set in the _db_engine_params dict on lines 32-35, so the settings are centralized.

Reply:
👍